pom.xml

```diff
@@ -71,6 +71,12 @@
 			<artifactId>liquibase-maven-plugin</artifactId>
 			<version>4.23.1</version>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
```

Comment:
In Spring Boot it's better to choose starters over concrete dependencies, so I suggest to use validation starter